pwiz_tools/Skyline/TestFunctional/PasteMoleculesTest.cs

```diff
@@ -123,6 +123,7 @@ private static string ToLocalText(string text)
         const string caffeineSMILES = "Cn1cnc2n(C)c(=O)n(C)c(=O)c12";
         const string caffeineKEGG = "C07481";
         const string caffeineFormula = "C8H10N4O2";
+        const string caffeineFormulaUnicode = "C\u2088H\u2081\u2080N\u2084O\u2082"; // Unicode subscripts

```

Comment:
I think a good idea to add something with a 9 in it to cover the full range from 0 to 9. I like that you have the H10 which covers both multiple characters and zero.